src/components/shared/Svg.css

```diff
@@ -75,6 +75,10 @@ html .arrow.expanded svg {
   transform: rotate(0deg);
 }
 
+.domain.debuggee {
+  fill: lightgreen;
```

Comment:
Actually, we don't need a color... We'll style it later :)

Reply:
We have `--green-50` that would work. http://design.firefox.com/photon/visuals/color.html#green

Reply:
The green in the mockup was a throw away color. I'd like to look at all the colors in tree holistically soon.